### PR TITLE
Theme Change and Reorganize

### PR DESCRIPTION
**Pull request by TheReconPilot**
- Theme changed to RTD-Sphinx
- All warnings sorted out
- Stray image files outside _static fixed
- Refactored heading levels
- Arranged content as per categories decided on the Sheet

### CODE REVIEW

File: source/miscellaneous.rst [hunk 1]
```diff
@@ -1,74 +1,62 @@
...
         outputName = args(0)
         }
         
 One could use the ``outputName`` and use it in the name of the csv file, for eg. 
 
-.. code::
+.. code-block:: scala
 
     SimulationListenerRegistry.register(
         new CsvOutputGenerator("C:\\Users\\username\\Documents\\"+outputName+".csv", new SEIROutputSpec(context))
```
Comment (dpcherian) [inline source/miscellaneous.rst:62]: Can you change this link, please? I know you didn't write it, but still. Just make it ``outputName+".csv"``, the rest of it is unnecessary. In general, I'd like all our paths to be from the root `BharatSim` folder. i.e. just `src/` or `target/` etc. with the forward slash.

Comment (dpcherian) [reply source/miscellaneous.rst:62]: (But don't worry about the slashes and other paths if it's too hard, @prathithbhargav and @SoumilK1 will have to make those changes.)

**Review by dpcherian — CHANGES_REQUESTED** [review]
Great work! Thanks! A couple of tiny changes to be made, before I merge it.

**Comment by TheReconPilot** [comment]
I rebased the updated main branch onto this local organize branch and pushed it, I suppose that is why Aleena's commits are showing in too?

Is there a better way to do it for future cases? Anyway,

- Organized Riz's and Aleena's content
- Included the recommendations

File: source/miscellaneous.rst (hunk 1 above)
Comment (TheReconPilot) [reply source/miscellaneous.rst]: I did something in 7226e9a. Is that what the path is supposed to be here? Just a bit confused.

File: source/firstprogram.rst [hunk 2]
```diff
@@ -140,7 +139,7 @@ Suppose you wanted your output to give you the numbers of susceptible, infected
   }
  
 * The first column (Step) stores the current time step, obtained using the ``context.getCurrentStep`` function
-* The next 3 columns store the number of Susceptible, Infected and Removed people respectively, by fetching the total number of ``Person`` nodes on the graph with the appropriate `infection status <#>`_.
```
[the remark this answers is not in the document]
Comment (TheReconPilot) [reply source/firstprogram.rst]: Put it back in 68b859e

**Review by dpcherian — APPROVED** [review]
Looks good! Merging changes. No clue why both of your commits are showing up together. But I don't think it matters anyway.

**Pull request merged**